src/platform/FreeBSD.cc

```diff
@@ -189,6 +219,19 @@ void FreeBSD::Execute(const CommandLine& c) const
 	}
 	setenv("LD_LIBRARY_PATH_FDS", libs.c_str(), 1);
 
+    int shm_FD = po_pack(map);
```

Comment:
Missing error check: what happens if `po_pack` returns -1?

Also, `shm_FD` should be `shmFD` or even just `shmfd`. Although the C-language libpreopen uses `under_score_names`, the C++ capsh code uses `camelCasedNames`.

Reply:
Also, we are using tabs rather than spaces for indentation in this file.